cluster/terraform/variables.tf

```diff
@@ -21,4 +22,5 @@ locals {
         "${var.resource_prefix}rg-tsc-aks-nodes-${var.environment}"
     )
     dns_prefix = "${var.resource_prefix}-tsc-${var.environment}"
+    cluster_cert_secret = "${var.environment}-development-teacherservices-cloud"
```

Comment:
This will only work for the dev clusters. Can we make it more generic so it works for the other clusters? test, platform-test, etc

Reply:
added cluster_kv_cert_suffix variable